static/app/views/replays/replays.tsx

```diff
@@ -90,7 +92,19 @@ class Replays extends React.Component<Props> {
   }
 
   render() {
-    const {organization} = this.props;
+    const {organization, location} = this.props;
+    const {query} = location;
+    const {cursor: _cursor, page: _page, ...currentQuery} = query;
+
+    const sort: {
+      field: '-timestamp';
+    } = {
```

Comment:
This type is incorrect, but also it's probably not needed

Reply:
changed the type for the sort to a string. Just in case.